packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -121,11 +122,13 @@ class Checkbox extends StatefulWidget {
   ///  * [MaterialState.hovered].
   ///  * [MaterialState.focused].
   ///  * [MaterialState.disabled].
+  /// {@endtemplate}
   ///
   /// When [value] is null and [tristate] is true, [MaterialState.selected] is
   /// included as a state.
   ///
-  /// If this property is null, [MaterialStateMouseCursor.clickable] will be used.
+  /// If null, then the value of [CheckboxThemeData.mouseCursor] is used. If
+  /// that is also null, then [MaterialStateMouseCursor.clickable] is used.
   final MouseCursor? mouseCursor;
```

Comment:
It would help to include a see-also for MaterialStateMouseCursor here. The existing API doc is a bit confusing, since it says `If [mouseCursor] is a [MaterialStateProperty<MouseCursor>]` which is true on the face of it but not helpful unless the developer is familiar  with  MaterialStateMouseCursor.

Reply:
I added a 'see also' section now to point to `MaterialStateMouseCursor`